File: Season 2/kata_10.rb
# codewars ruby implementation define function name in camelCase instead of snake_case
def decodeMorse(morse_code)
  morse_code
    .strip
    .split(/\s{3}/)
    .map { |word| MORSE_CODE.values_at(*word.split).join }
    .join(" ")
end

File: Season 2/kata_10.ts
import { MORSE_CODE } from './preloaded';
export function decodeMorse(morseCode: string): string {
  return morseCode
           .trim()
           .split(/\s{3}/)
           .map( (word) => word.split(" ").map( (char) => MORSE_CODE[char]).join("") )
           .join(" ")

}